test/index.html

```diff
@@ -161,6 +161,35 @@
         .blocklyToolboxCategoryContainer:focus-visible {
         outline: none;
       }
+
```

Comment:
Is it intentional to only put this CSS in the test page? Then users of the plugin will have to add this CSS to their applications themselves. I realized I don't know if we're following that approach with other style decisions in this plugin but it seems a bit strange if so since we want to provide good defaults. Does this still look good if you don't have this CSS?

Reply:
Yeah, I'll file an issue for that. We want it to be overridable because i'm assuming makecode has extensive custom styling, but it should look ok by default. maybe have a parameter to control whether to register any css or not. but maybe that's not necessary as you can just re-style it yourself if you don't like it.

Reply:
Sounds good, thanks! I'll merge as-is then.